### ISSUE TITLE
Remove "stat.moneycontrol.co.in" from blocklist.

### ISSUE THREAD

**Issue by Rexadev** [anudeepND/blacklist#174]
<div align="center">  
  <img width="280" height="70" src="https://raw.githubusercontent.com/anudeepND/blacklist/master/images/blacklist-logo.png" alt="logo" />
</div>

<h2 align="center">Issue Submit Form</h2>
<div align="center">
  Provide the following info properly, which will help me to resolve your issue quickly.
</div>

### Issue(s):
Type `x` in between `[ ]` and make sure there isn't any space between brackets. Example; for Your Selected Issue(s), type like this - `[x]` 
You can select more than one category of issues if you need to!

- [ ] Whitelist
- [ ] Blacklist
- [ ] App Broken
- [x] Website Broken
- [ ] Request
- [ ] Other Issue

```
moneycontrol.co.in
```
<hr>

### Domain(s):
If you are submitting this issue for whitelist/blacklist issue, send me the domain(s) for whitelisting/blacklisting here. Kindly use the **Code Tag** to prevent tracking.

- Domain(s):

```
stat.moneycontrol.co.in
```

<hr>

### Details:
You can attach any screen shot or log of the issue or advert, this will help to highlight it.

- Your Issue Detail:

**Comment by spirillen** [comment]
why? do you think this should be removed?

**Comment by Rexadev** [comment]
> why? do you think this should be removed?

Some things dont properly load and are not aligned

**Comment by anudeepND** [comment]
@Rexadev  @spirillen we had a similar issue with Moneycontrol, which was fixed in https://github.com/anudeepND/blacklist/issues/82 which I fixed by removing:
```
stat1.moneycontrol.com	
stat2.moneycontrol.com	
stat3.moneycontrol.com	
stat4.moneycontrol.com
```

And I couldn't find `stat.moneycontrol.co.in` in my logs while browsing the website and all the features such as charts, live chat, stock prices etc are working fine. Can you please provide more information @Rexadev ?

**Comment by Rexadev** [comment]
> @Rexadev @spirillen we had a similar issue with Moneycontrol, which was fixed in #82 which I fixed by removing:
> 
> ```
> stat1.moneycontrol.com	
> stat2.moneycontrol.com	
> stat3.moneycontrol.com	
> stat4.moneycontrol.com
> ```
> 
> And I couldn't find `stat.moneycontrol.co.in` in my logs while browsing the website and all the features such as charts, live chat, stock prices etc are working fine. Can you please provide more information @Rexadev ?

Sort feature breaks and table is mis-aligned

**Comment by anudeepND** [comment]
Sorry, I couldn't reproduce the issue can you post some screenshots?

> you are using .com and i suggested .co.in

`stat.moneycontrol.co.in` is not in my logs and I could not find it in dev console
